lib/app/features/feed/create_post/views/components/reply_input_field/reply_input_field.dart

```diff
@@ -36,7 +36,7 @@ class ReplyInputField extends HookConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final textEditorController = useQuillController();
 
-    final inputContainerKey = useRef(UniqueKey());
+    final inputContainerKey = useRef(GlobalKey());
```

Comment:
You can use `useRef` or `useMemoized` to keep the unique key alive after rebuild.